src/components/content/heroes/aurora/weather-easter-egg/rain.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flare } from '../flare';
+import PrecipitationEffect, { PrecipitationType } from './precipitation-effect';
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { DefaultFlares, DefaultFlaresDark } from '../default-flares';
+
+export const AuroraRainyFlareColor = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(180deg, #231f67 0%, #4d79ff45 90%);
+  position: absolute;
+`;
+
+interface RainProps {
+  amount: number;
+}
+
+export const Rain: FC<RainProps> = ({ amount }) => {
+  return (
+    <>
+      <AuroraRainyFlareColor />
+      <Flare
+        noAnimation={true}
+        size={970}
+        blur={125}
+        background={'linear-gradient(180deg, #4d79ff 0%, #4d79ff26 100%)'}
+        x={'38vw'}
+        y={'25vh'}
+      />
+      <Flare
+        noAnimation={true}
+        size={970}
+        background={'linear-gradient(180deg, #2a275f 0%, #535364 100%)'}
+        blur={125}
+        x={'30vw'}
+        y={'15vh'}
+      />
+      <Flare
+        noAnimation={true}
+        size={1094}
+        background={`linear-gradient(180deg, #150b28 0%, #23218326 100%)`}
+        blur={125}
+        x={'64vw'}
+        y={'64vh'}
+      />
+      <DefaultFlares />
+      <DefaultFlaresDark />
+      <PrecipitationEffect
+        dropCount={amount}
+        speed={1}
+        type={PrecipitationType.Rain}
+        speedDeviation={0.1}
```

Comment:
I understand this param, but the higher the value of derviation was when I tinkered around with it the lower the speed got. Then I realized that this is not the derivation but the speed being wrong. You forward it directly into the animation as the duration which gets you lower speeds for higher numbers. That's not what I expect. Good enough for here but something to think about 🙏